lib/__Test__/app.test.js

```diff
@@ -0,0 +1,93 @@
+const mongoose = require('mongoose');
+const { MongoMemoryServer } = require('mongodb-memory-server');
+const request = require('supertest');
+const app = require('../app');
+const TvNote = require('../models/TvNote');
+
+describe('app routes', () => {
+  const mongo = new MongoMemoryServer();
+  beforeAll(() => {
+    return mongo.getUri()
+      .then(uri => mongoose.connect(uri, {
+        useUnifiedTopology: true,
+        useNewUrlParser: true
+      }));
+  });
+  
+  beforeEach(() => {
+    return mongoose.connection.dropDatabase();
+  });
+  
+  afterAll(() => {
+    return mongoose.connection.close();
```

Comment:
Your tests aren't ending properly because you're not ending the MongoMemoryServer process.
If you include a .end in your test, that will fix the problem.

afterAll(() => {
    return mongoose.connection.close().
      then(mongo.stop());
  });